packages/flame_3d/lib/src/extensions/color.dart

```diff
@@ -4,9 +4,9 @@ import 'dart:ui';
 extension ColorExtension on Color {
   /// Returns a Float32List that represents the color as a vector.
   Float32List get storage => Float32List.fromList([
-        opacity,
-        red.toDouble() / 255,
-        green.toDouble() / 255,
-        blue.toDouble() / 255,
+        a,
+        r / 255,
+        g / 255,
+        b / 255,
```

Comment:
```suggestion
        a,
        r,
        g,
        b,
```

According to the docs these are already in normalized space

Reply:
derp, good catch